airflow/requirements.txt

```diff
@@ -1,2 +1,3 @@
-polygon-etl==0.1.8
 discord-webhook==0.14.0
+polygon-etl==0.1.9
+pytest==6.1.2
```

Comment:
Should we move this to dev dependencies?

Reply:
Moved `pytest` to `requirements_test.txt`, which is shared by airflow, cli and json validation tests.